styleguide/template.md

```diff
@@ -129,20 +131,30 @@ When linking to an API, you can use its unique identifier (UID) that is auto-gen
 
 You can use one of the following syntax:
 
-1. Markdown link: `[link_text](xref:UID)`
-2. Auto link: `<xref:UID>`
-3. Shorthand form: `@UID`
+1. Auto-link: `<xref:UID>` or `<xref:UID?displayProperty=fullName>`
+
+ The displayProperty query parameter produces a fully qualified link text. Default behavior is to produce a link text that shows only the member or type name.
+ 
+2. Markdown link: `[link_text](xref:UID)`
+
+ Use only when the auto-link behavior is not the desired display. 
```

Comment:
Can you clarify this? The meaning is unclear.

Reply:
Changed. See if it's better.